packages/rxtranscribe/src/operators/toAWS.js

```diff
@@ -0,0 +1,44 @@
+// copied from https://github.com/aws-samples/amazon-transcribe-websocket-static
```

Comment:
⬇️ This is basically how the AWS transcription pipeline works.  The most complicated piece is authentication and transforming the audio chunks into the format that AWS Transcribe accepts.  With a minor alteration, we should also be able to switch between the normal AWS Transcribe endpoint and their endpoint for Medical language.  Their medical API is too expensive to use for all of the audio we receive but it could be really useful for sanity checking, providing human review or seeing how good our model is versus Amazon's human augmented system (which I think uses human transcriptionists on the backend).